tests/SConscript

```diff
@@ -1,3 +1,9 @@
 # -*- python -*-
-from lsst.sconsUtils import scripts
+import os
+from lsst.sconsUtils import env, scripts
```

Comment:
Usually I am asked to order the imports such that standard modules and of non-standard imports are separated by a blank line. I'll leave it to you to decide whether you want to reorder the imports and introduce a blank line or not.